src/iOS.Core/Controllers/LockPasswordViewController.cs

```diff
@@ -258,6 +265,7 @@ private async void DoContinue(bool masterPassword = false)
             {
                 await _storageService.SaveAsync(Bit.Core.Constants.PasswordVerifiedAutofillKey, true);
             }
+            await EnableBiometrics();
```

Comment:
Can you elaborate on this change?

Reply:
Good callout, I created a method to check if biometrics have been validated yet so we can use SSO to enable them instead of MP if they are using key connector.

Regarding 2ecbc4b2c155fbd927af5558fb677b1d34f37821, I accidentally included an extra call in `CheckPasswordAsync()` that I removed in this commit (and did some housekeeping).

I admit this method will now also be called whenever biometrics are successful (unnecessarily), but I thought it was a little cleaner.